### ISSUE TITLE
Background Colour Appears Darker than Configured

### ISSUE THREAD

**Issue by MarkKoz** [jwilm/alacritty#2713]
Which operating system does the issue occur on?
Arch Linux

If on linux, are you using X11 or Wayland?
X11 + i3

If I configure the background to be `0x1c1c1c`,  the colour actually ends up being `0x141414` (tested with a colour picker). So to actually get it to be `0x1c1c1c` I have to set the colour to `0x282828`.

From `0x1c / 0x14` I calculated the factor to be `1.4`: 

```
0x1c * 1.4 = 0x28
```

**Comment by chrisduerr** [comment]
Do you have color profiles setup? Alacritty just renders out the colors exactly the same way they're specified.

If the color is specified correctly in the config and you're actually checking the color which you've specified, I'd say it's highly unlikely that the color would be different without some system configuration messing with color profiles.

**Comment by MarkKoz** [comment]
Not that I know of. Where would I check for that? Other things like polybar and i3's borders use correct colours.

**Comment by chrisduerr** [comment]
Could you post your config?

Also please make sure the correct config is loaded, you can check with `alacritty -v`.

**Comment by MarkKoz** [comment]
Turns out this issue was due to the background opacity being configured in alacritty. When I first started debugging this colour issue I did set it back to 1.0 but for some reason i3 had to be restarted for this change to take affect and I didn't realise that needed to happen. I discovered this when doing `alacritty -v` just now worked with the correct colours (before I was always launching alacritty via i3).
